PythonPractice/TicTacToe/tictactoe.py

```diff
@@ -0,0 +1,155 @@
+class SpaceTakenError(Exception):
+    pass
+
+class BoardTooSmallError(Exception):
+    pass
+
+class TicTacToe:
+
+    def __init__(self):
+        self.board = []
+        self.initialize_board()
+        self.playerO = 'O'
+        self.playerX = 'X'
+        self.player = self.playerX
+
+    def initialize_board(self):
+        """
+        Creates an empty board to be played on
+        """
+        # get board size from user
+        good_input = False
+        size = None
+        while not good_input:
+            try:
+                size = int(raw_input("What size board to do you want to play on?\n"))
+                if size < 3:
+                    raise BoardTooSmallError
+            except BoardTooSmallError:
+                print "Your board must be at least a 3x3 to play."
+            except ValueError:
+                print "Your board size must be an integer."
+            else:
+                good_input = True
```

Comment:
The use of a loop-breakout variable can be useful when you're trying to break out of multiple nested loops, but if you're using just one, you can change the loop to `while True:` and use `break` to break out of what would otherwise be an infinite loop.